src/cache/cache_las.c

```diff
@@ -710,6 +710,7 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
          */
         stop_ts = WT_TS_MAX;
         stop_txnid = WT_TXN_MAX;
+        onpage_upd_seen = list->onpage_upd.upd == NULL;
```

Comment:
@sulabhM Hey Sulabh, in our discussion about reverse delta, we decided we will always append the original value at the end of the update chain.  So can I change this to WT_ASSERT(session, list->onpage_upd.upd != NULL)?

Reply:
I think I figured it out. Here it should be set to onpage_upd_seen = false and I can do the assert in the do while loop.